UPDATING.md

```diff
@@ -40,6 +40,29 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Changes to BigQueryGetDataOperator
+`max_results` parameter type changed from `str` to `int` to reflect the correct data type 
+and `location` parameter removed as it is redundant.
```

Comment:
@nuclearpinguin Have you added this parameter recently? Can you give more explanations?

Reply:
PR created by @nuclearpinguin https://github.com/apache/airflow/pull/6020/files.
IMO this change seems to be against this PR and related JIRA's

Reply:
The location in used in the `get_tabledata` method which is used in execute.
https://github.com/apache/airflow/blob/65851f7a747a61ac486f18ec9b5dfcccb711027e/airflow/gcp/hooks/bigquery.py#L1489-L1508